src/commcare_cloud/ansible/roles/kafka/tasks/main.yml

```diff
@@ -9,8 +9,8 @@
 - name: Set JMX_PORT in /etc/environment
   lineinfile:
     dest: /etc/environment
-    line: 'JMX_PORT=9999'
-    state: present
+    line: 'JMX_PORT={{ KAFKA_JMX_PORT }}'
+    state: {{ 'present' if ansible_distribution_version == '14.04' else 'absent' }}
```

Comment:
Confirming this is supposed to be 14.04 although the PR description references 18.04?

Reply:
Yeah this is saying for 14.04 do it the old way (put this line in `/etc/environment`) and for 18.04 don't, put it in the systemd service file instead